### PR TITLE
Nest Init FDM Power

### PR DESCRIPTION
**Pull request by davidzwa**
- setup Nest + Better-SQLite3 (Auth, Users, migrate, fix tests, fix E2E)
- setup GHA
- setup Docker build
- setup Mend Renovate
- setup Mend Whitesource
- add allcontributors

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "matchPackageNames": ["node"],
+      "enabled": false
+    },
+    {
+      "matchDatasources": ["docker"],
+      "matchUpdateTypes": ["minor", "patch", "bump", "digest", "rollback"],
+      "enabled": true
+    },
+    {
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest", "lockFileMaintenance"],
```
Comment (mkevenaar) [inline renovate.json:29]: You mention lockFileMaintenance here, but you do not have that enabled

Comment (davidzwa) [reply renovate.json]: Much obliged! 👍🏼

**Review by davidzwa — COMMENTED** [review]
All checks out

**Comment by davidzwa** [comment]
@allcontributors add davidzwa for bugs maintenance code tests

**Comment by mkevenaar** [comment]
@davidzwa you forgot to setup Mend Whitesource

**Pull request merged**